MAUI/TreeMap/selection-and-highlights.md

```diff
@@ -9,11 +9,11 @@ documentation: ug
  
 # Selection in TreeMap control
 
-The Treemap control enables users to select specific groups or items for distinction, with the Tooltip feature providing additional information during interaction. However, when the SelectionMode is set to `None,` interaction with treemap items is disabled.
+The Treemap control enables users to select specific groups or items for distinction, with the tooltip feature providing additional information during interaction. However, when the `SelectionMode` is set to `None,` interaction with treemap items are disabled.
```

Comment:
The Treemap control enables users to select specific groups or items for distinction, with the tooltip feature providing additional information during interaction. However, when the `SelectionMode` is set to `None,` interaction with treemap items is disabled.

Reply:
Changed content